src/Tizen.NUI.Components/Theme/DefaultThemeCommon.cs

```diff
@@ -418,6 +418,31 @@ public Theme Create()
                 ActionCellPadding = new Size2D(40, 0),
             });
 
+            theme.AddStyleWithoutClone("Tizen.NUI.Components.Picker", new PickerStyle()
+            {
+                Size = new Size(160, 339),
+                ItemTextLabel = new TextLabelStyle()
+                {
+                    //FIXME: Should be check PointSize. given size from UX is too large.
+                    PixelSize = 32,
+                    VerticalAlignment = VerticalAlignment.Center,
+                    HorizontalAlignment = HorizontalAlignment.Center,
+                    Size = new Size(0,72),
+                    TextColor = new Selector<Color>()
+                    {
+                        Normal = new Color("#000C2BFF"),
+                    },
+                    BackgroundColor = Color.White,
+                },
+                Divider = new ViewStyle()
+                {
+                    Size = new Size(160, 2),
```

Comment:
How about changing this as follows to show Divider with correct size when Picker size is set by user?

SizeHeight = 2.0f,
WidthResizePolicy = ResizePolicyType.FillToParent,

Reply:
I agree with your comment.